Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM silintl/ubuntu:14.04
-MAINTAINER Phillip Shipley <phillip_shipley@sil.org>
+FROM alpine:3.5
 
-RUN apt-get update -y \
-    && apt-get install -y \
-        curl \
-        python-setuptools \
-        jq \
-    && easy_install pip \
-    && pip install awscli
+# Update APK cache
+RUN apk update
 
-COPY ecs-deploy /usr/local/bin/ecs-deploy
+# Install packges needed
+RUN apk add ca-certificates curl bash jq py2-pip && \
```

Comment:
Use `RUN apk add --no-cache ca-certificates curl bash jq py2-pip` to leave no package lists behind.